kroxylicious/src/main/java/io/kroxylicious/proxy/internal/KafkaProxyInitializer.java

```diff
@@ -172,7 +174,16 @@ private void addHandlers(SocketChannel ch, VirtualClusterBinding binding) {
 
             var filters = new ArrayList<>(filterChainFactory.createFilters());
             // Add internal filters.
-            filters.addAll(FilterAndInvoker.build(new BrokerAddressFilter(virtualCluster, endpointReconciler)));
+            var isUninitialized = false;
+            if (binding instanceof VirtualClusterBrokerBinding vcbb) {
+                isUninitialized = vcbb.uninitializedBrokers();
```

Comment:
naming: uninitializedBrokers  => upstreamKnown?

Reply:
definitely anything that removed the negative :) upstreamKnown sounds good. What about `upstreamAddressKnown` or `upstreamResolved`?